.travis.yml

```diff
@@ -133,14 +136,16 @@ after_success:
     fi
 
     export PACKAGING_REPO=elmaven-packaging
-    git clone https://github.com/ElucidataInc/elmaven-packaging.git
+    # TODO: reset this to master branch after merge
```

Comment:
Has the packaging PR been merged? please update

Reply:
Yes it has. I will remove the comment.